### DISCUSSION TITLE
Chat Room for dotnet/TorchSharp

### DISCUSSION THREAD

**Discussion by NiklasGustafsson** [xamarin/TorchSharp#727] [Announcements]
I just created a new gitter room for chatting about TorchSharp: 

https://gitter.im/dotnet/TorchSharp

**Comment by GeorgeS2019** [comment]
[![Join the chat at https://gitter.im/dotnet/TorchSharp](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/dotnet/TorchSharp?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)

`[![Join the chat at https://gitter.im/dotnet/TorchSharp](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/dotnet/TorchSharp?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)`
